_posts/2015/2015-12-07-codemotion-madrid.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Codemotion 2015 - One ePagees was there!"
+date: "2015-12-07 12:00:00"
+icon: search
+categories: conferences
+authors: ["Ángel García"]
+---
+The Codemotion conference took place in Madrid on November 28 and 29. ePages was present there. With more than 1900 software enthusiasts, Codemotion is the biggest IT event in Spain. This conference was sponsored by companies such as IBM, Microsoft, Arsys, Mozilla, Google... Two days fulled of talks from 12 different tracks, where each track
```

Comment:
ePages was present there. > I would leave that out as it is mentioned in the title already.